### DISCUSSION TITLE
Performance issue with the line graph.

### DISCUSSION THREAD

**Discussion by GabrielCosta16** [imaNNeo/fl_chart#1550] [Q&A]
Good morning, I'm participating in a college project where I'm going to develop a prototype of a graphics panel. Using the lineChart graph, as the y-axis values ​​increase, the graph becomes slower and when the y-axis values ​​exceed 6 digits, the page freezes. Would you have any idea how I could improve performance?

**Comment by imaNNeo** [comment]
Hi, we render all the points (which is not a good idea), then if you have a lot of points, performance decreases.
Later we will have some internal optimization of the data. But for now, you can do it by yourself. You can optimize the data and remove some points that don't affect the result.
